### DISCUSSION TITLE
FMU: Windows to Linux?

### DISCUSSION THREAD

**Discussion by samiit** [ThummeTo/FMI.jl#48] [Q&A] [answered]
Thank you for this helpful package.

I am sorry to ask this question here, but I request for some pointers. We have Matlab/Simulink installed on a Windows machine, and when exporting the FMUs, we are not able to load it on a Linux machine. How do you suggest loading such an FMUs using FMI.jl?

Thanks,
Sam

**Comment by ThummeTo** [comment, marked as answer]
Dear Sam,

I moved this to "Discussions". Many modeling tools only export a FMU for the current platform, meaning tools that are running under Windows generate Windows-FMUs and tools that run under Linux export Linux-FMUs (but there are exceptions). Personally, if I run into this issue, I enable C-Code-export when exporting FMUs, so it is possible to build the FMU for every system (in theory) from source code.

But in your special case this might be even more complicated. FMUs exported from Matlab/Simulink often have further dependencies to Matlab-Libraries (DLLs), which need to be resolved to the corresponding Linux Libraries (if they exist on the system). Because this question (if I understand correctly) is more related to the FMI-export in Simulink than to the FMI-import with FMI.jl, I suggest to contact the Matlab/Simulink support for this problem. Could you provide the error message? That would help a lot.

Best regards!
